recipe/meta.yaml

```diff
@@ -35,10 +35,11 @@ test:
 
 about:
   home: https://github.com/tqdm/tqdm
-  license: Mozilla Public License 2.0 (MPL 2.0) or MIT License
+  license: MPLv2.0, MIT
   summary: 'A Fast, Extensible Progress Meter'
 
 extra:
   recipe-maintainers:
+    - casperdcl
```

Comment:
@casperdcl we do not handle organizations in our bot scripts making it hard to do what you ask. Is it complicated for you to add your crew via individual GitHub handles?